README.md

```diff
@@ -65,7 +65,7 @@ You have two ways to install Crowdin Android SDK.
 
    ```groovy
    dependencies {
-       implementation 'com.github.crowdin.mobile-sdk-android:sdk:1.4.1'
+       implementation 'com.github.crowdin.mobile-sdk-android:sdk:1.4.3'
```

Comment:
@MykhailoNester why 1.4.3?

Reply:
updated to 1.4.2. Was changed by mistake in two commits.